ibm/validate/validators.go

```diff
@@ -120,6 +120,22 @@ func ValidateAllowedIntValues(is []int) schema.SchemaValidateFunc {
 	}
 }
 
+func ValidateBucketIdRegex(regex string) schema.SchemaValidateFunc {
```

Comment:
where is this ValidateBucketIdRegex used?

Reply:
As per earlier changes I was using. Removed now.